addons/website_payment/static/tests/tours/donation.js

```diff
@@ -0,0 +1,122 @@
+        },
+        {
+            content: "Donate with custom amount set",
+            trigger: ".s_donation_donate_btn",
+            run: "click"
+        },
+        {
+            content: "Check if custom amount radio input is selected and value is 55",
+            trigger: "input#other_amount",
+            run: function(action) {
+                const input = action.tip_widget.$anchor[0];
+                const inputValue = input.nextElementSibling.children[0].value;
+                if (!input.checked || inputValue !== "55.0") {
+                    console.error("The custom amount radio input is not selected or the value is not 55.");
+                }
+            }
+        },
+        {
+            content: "Select the amount of 25",
+            trigger: "input#amount_1",
+            run: "click"
+        },
+        {
+            content: "Verify that only one input is checked",
+            trigger: "input#amount_1",
+            run: function(action) {
+                const amount1 = action.tip_widget.$anchor[0];
+                const amountDiv = amount1.parentElement.parentElement;
+                const radiosInputs = amountDiv.querySelectorAll("input[type=radio]");
+                const selectedInputs = Array.from(radiosInputs).filter(radio => radio.checked);
+
+                if (selectedInputs.length !== 1) {
+                    console.error("More than one input is checked.");
+                }
+            }
+        },
+        {
+            content: "Change custom amount to 67",
+            trigger: "input#other_amount",
+            run: function(action) {
+                const input = action.tip_widget.$anchor[0];
+                const customInput = input.nextElementSibling.children[0];
+                customInput.value = "67";
+                customInput.dispatchEvent(new Event("input", { bubbles: true }));
+                customInput.dispatchEvent(new Event("change", { bubbles: true }));
+            }
+        },
+        {
+            content: "Select the custom amount radio button",
+            trigger: "input#other_amount",
+            run: "click"
+        },
+        {
+            content: "Submit the donation form",
+            trigger: "button[name='o_payment_submit_button'",
+            run: "click"
+        },
+        {
+            content: "Verify that the amount displayed is 67",
+            trigger: "span.oe_currency_value",
+            run: function(action) {
+                const amountElement = action.tip_widget.$anchor[0];
+                if (!amountElement || amountElement.textContent.trim() !== "67.00") {
+                    console.error("The amount displayed is not 67.00.");
+                }
+            },
+            timeout: 10000
+        },
+        {
+            content: "Go to /?enable_editor=1",
+            trigger: "body",
+            run: () => {
+                location.href = "/?enable_editor=1";
+            }
+        },
```

Comment:
Try to navigate more "naturally" like a user would:
- click on a link to return to the home page
- use `wTourUtils.clickOnEdit(),` to return to edit mode

Reply:
This is just to remove the snippet, I think it makes the test more robust given the fact that I start the test using the same URL. Let me know what you think.

Reply:
You could also not do the cleanup: the DB transaction is rolled back at the end of the test anyway - so everything that was added will not be there for the next test.